cmd/publish-plugin.go

```diff
@@ -0,0 +1,132 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"net/url"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/go-flutter-desktop/hover/internal/log"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(publishPluginCmd)
+}
+
+var publishPluginCmd = &cobra.Command{
+	Use:   "publish-plugin",
+	Short: "Assert that your go-flutter plugin can be pushed as golang module in your github repo.",
```

Comment:
Short description is misleading.  We don't just assert, we actually run the commands to publish (after confirmation).